src/handlers/blame.rs

```diff
@@ -233,15 +233,15 @@ $
 pub fn parse_git_blame_line<'a>(line: &'a str, timestamp_format: &str) -> Option<BlameLine<'a>> {
     let caps = BLAME_LINE_REGEX.captures(line)?;
 
-    let commit = caps.get(1).unwrap().as_str();
-    let author = caps.get(2).unwrap().as_str();
-    let timestamp = caps.get(3).unwrap().as_str();
+    let commit = caps.get(1)?.as_str();
+    let author = caps.get(2)?.as_str();
+    let timestamp = caps.get(3)?.as_str();
 
     let time = DateTime::parse_from_str(timestamp, timestamp_format).ok()?;
 
-    let line_number = caps.get(4).unwrap().as_str().parse::<usize>().ok()?;
+    let line_number = caps.get(4)?.as_str().parse::<usize>().ok()?;
 
-    let code = caps.get(5).unwrap().as_str();
```

Comment:
`unwrap()` here is actually okay because the regex is know at compile time. If it is changed but not the corresponding code here then it _should_ crash for the developer and not silently fail and return None.
Similarly for the max line numbers above.

Reply:
> `unwrap()` here is actually okay because the regex is know at compile time. If it is changed but not the corresponding code here then it _should_ crash for the developer and not silently fail and return None. Similarly for the max line numbers above.

should we reverting this changes?

Reply:
Yes, I think the only bit of the PR that arguably should be retained is not using `else` after the early return in `src/align.rs` -- I do slightly prefer that. But beyond that, there are lots of bugs and feature requests to work on if you've got time! And any performance improvements backed by profiling of course would be welcome.

Reply:
We could create and tag issues as **🟣 good first issue** for that.

Reply:
Thanks for the review, corrected